src/IQFeed.CSharpApiClient/Common/FieldParser.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Globalization;
+
+namespace IQFeed.CSharpApiClient.Common
```

Comment:
maybe you could move this closer to your implementation since its the only consumer for now?

Reply:
nevermind.